app/controllers/support_controller.rb

```diff
@@ -0,0 +1,18 @@
+class SupportController < LoggedAreaController
```

Comment:
The support page is a static page that will sit at the root of the site (`/support`), so I think it should be a controller action in here: https://github.com/alphagov/datagovuk_find/blob/master/app/controllers/pages_controller.rb.

Reply:
The resource we are dealing with here is a `Ticket`, so it makes sense to have a `TicketsController`.

Reply:
Interesting article related to this topic: http://jeromedalbert.com/how-dhh-organizes-his-rails-controllers/

Reply:
Thanks @laurentqro 👍